net/vnstat/files/vnstat.init

```diff
@@ -72,6 +72,10 @@ start_service() {
 	procd_set_param stdout 1
 	procd_set_param stderr 1
 	procd_set_param command /usr/sbin/vnstatd --nodaemon
+	procd_set_param respawn
+	procd_append_param respawn 3600
```

Comment:
good catch - thanks
I'll fix this once we decide how to proceed with the three separate (partially 
conflicting) pull-requests